plugins/kotlin/idea/tests/testData/formatter/trailingComma/whenEntry/WhenEntry.after.kt

```diff
@@ -50,17 +48,16 @@ fun foo(x: Any) {
 
     when (x) {
         1,
-        -> {
+            -> {
 
-        }
+            }
 
         else -> println(3)
     }
 
     when (x) {
         1, 2,
-        3, /**/
-        -> {
+        3 /**/ -> {
```

Comment:
The same as above, the trailing comma is missing

Reply:
And this one